### DISCUSSION TITLE
For all looking to run on MAC

### DISCUSSION THREAD

**Discussion by worldoptimizer** [acheong08/ChatGPT#69] [Ideas]
I came across some limitations like the possibility to scale as big as needed etc. I ended up using the Menubar X app on Mac that is (exclusive to that platform). It allows for arbitrary apps to be placed in the menu bar. I have ChatGPT and Open AI Playground up there.

**Comment by acheong08** [comment]
Should be a discussion

**Reply by worldoptimizer** [reply]
Oh, sorry. I was had the issue that I could see the entire answer as the font was too big (height). Anyhow, your version is cross-platform, so it benefits many more. I was just giving a hint on how I solved it for now. Keep up the good work.

**Reply by acheong08** [reply]
Could you elaborate? 

> font was too big 

I don't have a specific font set

**Reply by acheong08** [reply]
Where are you running it from? Terminal? If it is, you can just scroll up

**Reply by worldoptimizer** [reply]
I am embarrassed now... wrong project. I have too many tabs open.
I wanted to comment here: https://github.com/vincelwt/chatgpt-mac

Really sorry.
